### PR TITLE
[Clojure] Add donkey

### PR DESCRIPTION
**Pull request by whiplash**
Hi @yaronel,

Would you mind to add (or consent that we add) [donkey](https://github.com/AppsFlyer/donkey) in `here` ?

------------
This project compare performance based on 3 routes :

+ `GET` `/`, **SHOULD** return a successful status code and an empty string
+ `GET` `/user/:id`, **SHOULD** return a successful status code and the id
+ `POST` `/user`, **SHOULD** return a successful status code and an empty string

Regards,

### CODE REVIEW

**Comment by yaronel** [comment]
Hi. 
Yeah, go ahead and add it. 
just please add the command line arguments as they appear here:
https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Clojure/donkey/donkey.dockerfile

**Comment by waghanza** [comment]
You mean https://github.com/TechEmpower/FrameworkBenchmarks/blob/6d43ad15d76b3dc1c51bcf988b30d86fd876fd22/frameworks/Clojure/donkey/donkey.dockerfile#L12, right ?

`vertx` options are OK, but I'm not sure about `Xsm2G`. is it for memory limit ?

**Comment by whiplash** [comment]
`Xms` specifies the initial memory allocation size

**Comment by mfirry** [comment]
Not sure `UseStringDeduplication` and `UseNUMA` will make much difference in this case.

**Comment by yaronel** [comment]
The vertx options are the most important. 
Setting the min and max memory to the same value avoids committing more memory during runtime.

File: clojure/donkey/config.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+framework:
+  website: https://github.com/AppsFlyer/donkey
```
Comment (waghanza) [inline clojure/donkey/config.yaml:2]: clearer 😛 

```suggestion
  github: AppsFlyer/donkey
```

File: clojure/donkey/config.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  version: 0.4.0
```
Comment (waghanza) [inline clojure/donkey/config.yaml:3]: better
```suggestion
  version: 0.4
```

**Review by waghanza — CHANGES_REQUESTED** [review]
minor typos

File: clojure/donkey/project.clj [hunk 3]
```diff
@@ -0,0 +1,8 @@
+(defproject the-benchmarker-donkey "0.1.0-SNAPSHOT"
+  :dependencies [[org.clojure/clojure "1.10.1"]
+                 [com.appsflyer/donkey "0.4.0"]]
```
Comment (waghanza) [inline clojure/donkey/project.clj:3]: is it required to pin the exactas  version in `clojure` ? could not we use a range, like `[0.4,0.5)` @yaronel

Comment (whiplash) [reply clojure/donkey/project.clj:3]: Not sure how leiningen handles ranges. Will investigate.

**Review by waghanza — APPROVED** [review]

**Pull request merged**